### ISSUE TITLE
Receive email notifications when there are new participants

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#771]
# TL;DR
Receive email notifications when there are new participants

# Narrative
It would be nice for researchers to receive an email notification when there are new participants. This can be one email per participant, or an email digest for participants completing a given study on a given day. The email can include participants who only complete consent forms (without completing the study) and participants who complete the whole study.

One reason might be that it's helpful for researchers to know promptly when they have new participants, so that they don't miss out on paying participants on time. 

# Acceptance Criteria
- [ ] Researcher reliably received email for each response on their studies. 
- [ ] Tests for new feature
- [ ] QA on new feature

**Comment by bleonar5** [comment]
this feature was requested again on April 4th https://lookit-mit.slack.com/archives/C01AQ9961EW/p1680635324834819
